standard/links/commons/links.lua

```diff
@@ -189,19 +193,14 @@ function Links.makeFullLink(platform, id, variant)
 		return ''
 	end
 
-	local prefix
-	if _PREFIXES[variant] then
-		prefix = _PREFIXES[variant][platform]
-	end
-
-	if not prefix then
-		prefix = _PREFIXES[platform]
-	end
+	local prefixData = _PREFIXES[platform]
 
-	if prefix == nil then
+	if not prefixData then
```

Comment:
Is there a particular reason why we do check prefixData, but not suffixData?

Reply:
the current implementation already does it that way
additionally so far there is no suffix without an accompanying prefix
while most prefixes do not have a suffix